test/offsetarray.jl

```diff
@@ -268,8 +268,7 @@ am = map(identity, a)
 # other functions
 v = OffsetArray(v0, (-3,))
 @test_approx_eq v v
-@test parent(v') == v0'
```

Comment:
is there an equivalent existing test for 2d offset arrays that your work doesn't touch?